### DISCUSSION TITLE
React Native - redirect user to login screen after verify email link

### DISCUSSION THREAD

**Discussion by ahmedabdelhafez** [supabase/supabase#7331] [Questions]
**Hello Supabase Team,**
1-now i develop an application with react naative and i want to redirect user to login screen after click the verify link on email.
2- and i want to know also how to verify signed up user with verification code instead of link

thanks

**Comment by samymih** [comment]
Any updates ?

**Comment by javierguzman** [comment]
Has any of you managed to do this? I want to do the same. Thank you in advance and regards

**Comment by VolodymyrDubnytskyi** [comment]
+ 1

**Comment by panda8z** [comment]
Any  updates here?
